lib/hades_web/controllers/user_controller.ex

```diff
@@ -12,4 +12,12 @@ defmodule HadesWeb.UserController do
       |> render("show.json", user: user)
     end
   end
+
+  def update_user_status(conn, %{"user" => user_params}) do
+    current_user = Hades.Guardian.Plug.current_resource(conn)
```

Comment:
Also, in a next PR we could extract this line to a separated module since we are using it a lot, and when using this new module and we inject the current user as a param in all actions. This can avoid future issues when `Guard` change their DSL again. Just a minor though!!

Reply:
That's an interesting idea - perhaps we can make a wrapper around it.
I think injecting it to all actions won't work on the create_user though because you won't have a user to get the current one.

Reply:
Yes, we can do it. Would be very welcome if you could show us some Guardian tricks :)